### PR TITLE
fill out setup.py

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
-    'psycopg2',
-]
-
-setuptools.setup(
-    name='cockroach',
-    version='0.1',
-    packages=setuptools.find_packages(),
-    install_requires=install_requires,
+setup(
+    name='django-cockroach',
```
Comment (timgraham) [inline setup.py]: To be confirmed: https://github.com/cockroachdb/cockroach-django/issues/87

File: setup.py [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+    version=__import__('cockroach').__version__,
+    python_requires='>=3.5',
+    url='https://github.com/cockroachdb/cockroach-django',
+    author='Cockroach Labs',
+    author_email='????',
```
Comment (timgraham) [inline setup.py]: Author email needed

File: setup.py [hunk 3]
```diff
@@ -1,12 +1,31 @@
...
-    version='0.1',
-    packages=setuptools.find_packages(),
-    install_requires=install_requires,
+setup(
+    name='django-cockroachdb',
+    version=__import__('django_cockroachdb').__version__,
+    python_requires='>=3.5',
+    url='https://github.com/cockroachdb/django-cockroachdb',
+    maintainer='Cockroach Labs',
+    maintainer_email='python@cockroachlabs.com',
```
Comment (timgraham) [inline setup.py]: This email to be confirmed (suggested to be created as a Google group by Ben in the slack channel).

Comment (rafiss) [reply setup.py]: I'm getting the group made now; I'll post back when it's done.

**Pull request merged**